client/galaxy/scripts/mvc/library/library-dataset-view.js

```diff
@@ -182,22 +177,12 @@ define(
                     var inputs = "";
                     $.each(data.split("&"), function() {
                         var pair = this.split("=");
-                        inputs +=
-                            '<input type="hidden" name="' +
-                            pair[0] +
-                            '" value="' +
-                            pair[1] +
-                            '" />';
+                        inputs += `<input type="hidden" name="${pair[0]}" value="${pair[1]}" />`;
                     });
                     //send request
                     $(
-                        '<form action="' +
-                            url +
-                            '" method="' +
-                            (method || "post") +
-                            '">' +
-                            inputs +
-                            "</form>"
+                        `<form action="${url}" method="${method ||
+                            "post"}">${inputs}</form>`
```

Comment:
Is this line break intended?

Reply:
@nsoranzo It's all automatic formatting, nothing here was manually done.  The break is due to the 80 character line length setting for prettier.